In [1]:
%%html
<style>
img[alt$=quinientos] {
    width: 500px;
}

img[alt$=cuatrocientos] {
    width: 400px;
}
</style>

In [2]:
# Para los calculos
import numpy as np


import scipy                  # http://scipy.org/
from scipy import constants

import matplotlib.pyplot as plt
from IPython.display import Image  
%matplotlib inline
params = {'legend.fontsize': 24,
          'figure.figsize': (12, 8),
         'axes.labelsize': 24,
         'axes.titlesize': 24,
         'xtick.labelsize':24,
         'ytick.labelsize':24}
plt.rcParams.update(params) 




# Factor de ruido en dispositivos conectados en cascada.

La fórmula de Friis se utiliza para calcular el factor de ruido total de una cascada de etapas, cada una con su propio factor de ruido y ganancia de potencia (suponiendo que las impedancias se combinan en cada etapa). 

![10-cuatrocientos](fig/noiseFriis.png)


El factor de ruido total se puede utilizar para calcular la cifra de ruido total. 

El factor de ruido total se da como

$$F = F_1 + \frac{F_2-1}{G_1} + \frac{F_3-1}{G_1 \cdot G_2} + ... + \frac{F_n -1}{G_1\cdot G_2 ... \cdot G_{n-1}}$$

#### Demostración

Suponiendo el sistema que se muestra en la figura, la potencia de señal a la salida se calcula como (suponiendo que el calculo de potencia tiene en cuenta la impedancia de entrada y de salida de las distintas etapas del sistema):

$$S_o = S_i \cdot G_1 \cdot G_2 \cdot G3$$

La potencia de ruido N_o en este caso sera:

$$N_o = N_i \cdot G_1 \cdot G_2 \cdot G_3 + N_{a1} \cdot G_2 \cdot G_3 + N_{a2} \cdot G3 + N_{a3}$$ 

El factor de ruido entonces:

$$F = \frac{\frac{S_i}{N_i}}{\frac{S_o}{N_o}}$$

$$F = \frac{S_i \cdot (N_i \cdot G_1 \cdot G_2 \cdot G_3 + N_{a1} \cdot G_2 \cdot G_3 + N_{a2} \cdot G3 + N_{a3}  ) }{N_i \cdot (S_i \cdot G_1 \cdot G_2 \cdot G3 )} $$

$$F = F_1 + \frac{F_2-1}{G_1} + \frac{F_3-1}{G_1 \cdot G_2} + ... + \frac{F_n -1}{G_1\cdot G_2 ... \cdot G_{n-1}}$$



### Factor de ruido de una red con pérdidas.

A temperatura ambiente, el factor de ruido de una red con pérdidas es igual a su pérdida de potencia.

La potencia de ruido en la entrada del atenuador debido a una fuente perfectamente adaptada a la temperatura ($T$) viene dada por

$$N_i = k_B \cdot T \cdot B$$


La densidad espectral de potencia de ruido de entrada es

$$\frac{N_i}{B}  = k_B \cdot T$$

Cuando una señal ruidosa pasa a través del atenuador, la potencia de ruido se atenúa de la misma manera que la potencia de la señal ($G = \frac{1}{L}$). Entonces, la densidad espectral de potencia del ruido en la salida es

$$\frac{N_o}{B} = \frac{N_i}{L \cdot B} = \frac{k_B  \cdot T}{L \cdot B}$$

donde $L = 10^{L_{dB}/10}$

Solo sale una parte del ruido, y el resto se disipa en el atenuador. Bajo el supuesto de que el atenuador está en equilibrio térmico, no sale calor por el atenuador debido al gradiente de temperatura.

El calor disipado en el atenuador es equivalente al exceso de potencia de ruido (N_e) en el atenuador.

$$\frac{N_e}{B} =  \frac{N_i}{B} - \frac{N_o}{B} =  \frac{N_i}{B} \cdot ( 1 - \frac{1}{L}  )$$

Factor de ruido del atenuador,

$$F = 1 + \frac{   N_e }{ G \cdot N_i}  =$$

$$F = 1 + \frac{ N_i \cdot ( 1 - \frac{1}{L}  )   }{ G \cdot N_i}  =$$

$$F = 1 + L \cdot ( 1 - \frac{1}{L}  )$$

$$F =  L$$ 

# Ancho de banda equivalente

Hasta el momento el analisis realizado permite el calculo de la densidad espectral de ruido. Para conocer la potenica de ruido es necesario conocer el ancho de banda de los circuitos. 

A continuación se analiza el ancho de banda de circuitos con impedacias complejas y luego, circuitos sintonizados. 

## Impedancias complejas

Nyquist determinó que la densidad espectral de ruido generado en una impedancia solo depende del valor resistivo de la impedancia (Parte real). El valor de tensión cuadrático medio en bornes de la impedancia se calcula integrando la densidad espectral sobre el ancho de banda de interés. El cálculo de la tensión de ruido tiene la siguiente expresión.


$${\bar{v_{n}^2}} = {4 k_B T \int_{B} R(f) df}$$


La expresión anterior equivale tener infinitas resistencias de  valor $R(f)$ que solo aportan ruido en un diferencial tensión cuadrática media $4 k_B T R(f) df$ centrado en $f$.

Para fijar conocimientos, consideremos un resistor en paralelo con un capacitor.



El valor de impedancia en bornes se calcula:

$$Z(f) = \frac{R}{1+j\omega R C}$$


Donde


$$Re [Z(f)] = R(f) = \frac{R}{ 1+ (\omega R C)^2 }$$


Por lo tanto, para conocer el valor cuadrático medio de la tensión de ruido a la salida debemos resolver la siguiente integral

$${\bar{v_{n}^2}} = {4 k_B T R \int_{B} \frac{1}{ 1+ (2 \pi f R C)^2} df}$$


Observar que la constante que multiplica a la integral tiene unidades de $V^2/Hz$, por lo tanto, la integral debe tener unidades de $Hz$. Más adelante veremos que esta integral determina el ancho de banda de ruido equivalente y esta dado pros la respuesta en frecuencia del circuito. Para resolver la siguiente integral, debemos conocer el ancho de banda sobre el que queremos medir el ruido, supongamos que poseemos un instrumento de ancho de banda infinito y queremos verificar el nivel de ruido,la integral que debemos resolver es:

$${\bar{v_{n}^2}} = {4 k_B T R \int_{0}^{\infty} \frac{df}{ 1+ (2 \pi f R C)^2} }$$

Recurriendo a una tabla de integrales entonces:

$${\bar{v_{n}^2}} = \frac{k_B T}{C}$$

Sabiendo que para un filtro RC, la frecuencia de corte se halla en $f_{-3db}=\frac{1}{2 \pi R C}$. La tensión cuadrática media de ruido toma la expresión:

$${\bar{v_{n}^2}} = {4 k_B T R (\frac{\pi}{2} f_{-3db})}$$

Donde el término entre paréntesis es el denominado ancho de banda equivalente de ruido.


In [5]:
# Ejemplo de calculo de potencia de ruido
# RC en paralelo
## Calculo con Jupyter

kb = constants.value('Boltzmann constant')
T  = 300  # K
R  = 1e3  # ohm 
C  = 1e-9 # F


# Potencia de ruido
Prms  = (kb*T/C)/R  # [V/sqrt(Hz)]

print('Potencia de ruido sobre el resistor: {:1.2e} W'.format(((Prms))))

Vrms  = (Prms*R)**(1/2)
print('Valor cuadratico medio de la tensión sobre el resistor: {:1.2e} V'.format(Vrms) ) 


Potencia de ruido sobre el resistor: 4.14e-15 W
Valor cuadratico medio de la tensión sobre el resistor: 2.04e-06 V


### Medición con LTspice

![11-cuatrocientos](fig/capacitor.png)

Valor cuadratico medio de la tensión sobre el resistor: $2.04e-06 V$

Potencia de ruido sobre el resistor: 

$$P = 4.14 \times 10^{-15} W$$


# Ancho de banda equivalente en circuitos sintonizados

Tanto el ruido térmico como el de disparo poseen densidades espectrales planas. Se define como ancho de banda equivalente de ruido de un sistema, al ancho de banda que debería tener un dispositivo ideal para producir en la salida la misma potencia de ruido. Esto re representa gráficamente en la siguiente imagen.

![12-cuatrocientos](fig/eqnb.png)

La potencia de ruido a la salida del dispositivo puede calcularse integrando la densidad espectral de potencia de ruido a la salida $N_{noise}*|H(f)|^2$ para todas las frecuencias, donde $|H(f)|^2$ es la ganancia de potencia en función de la frecuencia del dispositivo (puede ser un amplificador, un filtro o un mezclador). Esta potencia tiene que coincidir con la potencia a la salida de un filtro ideal de ancho de banda $B_{eq}$ y ganancia igual a la ganancia en frecuencia central $f_{c}$. Esto se expresa matemáticamente de esta forma:

$P_{out}=\int_{-\infty}^{\infty} N_{0} \cdot |H(f)|^2 df= N_{0} \cdot |H(f_{c})|^2\int_{-B_{eq}/2}^{B_{eq}/2} df$

Donde $N_{noise}$ es la densidad espectral de ruido y una constante. Resolviendo la integral en el termino derecho se obtiene:

$N_{0} \int_{-\infty}^{\infty} |H(f)|^2 df= N_{0} \cdot |H(f_{c})|^2 B_{eq}$

Esto demuestra que el ancho de banda se calcula de la siguiente forma:

$B_{eq}=\frac{\int_{-\infty}^{\infty} |H(f)|^2 df}{|H(f_{c})|^2}=\int_{-\infty}^{\infty} \frac{|H(f)|^2}{|H(f_{c})|^2} df =\int_{-\infty}^{\infty} \bar{|H(f)|^2} df$

### Ancho de banda equivalente de  un simple sintonizado

Para demostrar la aplicación de este concepto,calculemos el ancho e banda equivalente de ruido
de un amplificador sintonizado. El mismo tiene la siguiente respuesta en frecuencia normalizada:

$$ \bar{|H(f_{c})|^2}=\frac{1}{1+\chi^2}$$

Donde: $\chi^2=\frac{2 Q}{f_{c}} \cdot (f-f_{c})$, entonces $df= \frac{f_{c}}{2Q}d\chi$. El ancho de banda equivalente se calcula de esta forma

$$ B_{eq}=\int_{-\infty}^{\infty} \frac{1}{1+\chi^2} \frac{f_{c}}{2Q}d\chi $$

Este resultado es el mismo que el del filtro RC calculado anteriormente. El ancho de banda equivalente es el ancho de banda de 3dB por $\pi/2$

$$ B_{eq}=\frac{\pi}{2} \cdot f_{3dB}=\frac{\pi}{2}\frac{f_{c}}{Q}$$


###  Ancho de banda equivalente de dos simples sintonizados sincronicos y de igual Q

$$\bar{|H(f_{c})|^2}=\frac{1}{(1+\chi^2)^2}$$

Donde: $\chi^2=\frac{2 Q}{f_{c}} \cdot (f-f_{c})$, entonces $df= \frac{f_{c}}{2Q}d\chi$. El ancho de banda equivalente se calcula de esta forma

$$B_{eq}=\int_{-\infty}^{\infty} \frac{1}{(1+\chi^2)^2} \frac{f_{c}}{2Q}d\chi$$

Este resultado es el mismo que el del filtro RC calculado anteriormente. El ancho de banda equivalente es el ancho de banda de 3dB por $\pi/2$

$$B_{eq}=\frac{\pi}{4} \cdot f_{3dB}=\frac{\pi}{4}\frac{f_{c}}{Q}$$


###  Ancho de banda equivalente de un doble sintonizados


En el caso de un amplificador doble sintonizado, la transferencia normalizada es:

$$\bar{|H(f_{c})|^2}=  \frac{1}{1+\frac{\chi^4}{4}}$$

$$B_{eq}=\frac{\pi}{2} \cdot f_{3dB}=\frac{\pi}{2}\frac{f_{c}}{Q}$$


## Ejemplo Simple  Sintonizado empleando transistor con cifra de ruido $ NF = 2 dB $.

Calcular el ancho de banda equivalente de un amplificador simple sintonizado. Calcular para máxima transferencia de energía. Suponer que el aporte de la fuente de ruido en exceso se encuentra en la entrada.

Datos:
$f_o  = 100 MHz$

$Q_c  = 40$

$Q_o = inf$  

Transistor: 
$y11= 1.0 mS$,
$y12= 0.0 mS$, 
$y21= 30.0 mS$, 
$y22= 0.1 mS$.

In [6]:
## Ejemplo con calculo en Python
# simple sintonizado de 100MHz
# Frecuencia de operación
fo = 100e6
wo= 2*np.pi*fo

kb = constants.value('Boltzmann constant')
# Temperatura en Kelvin
T  = 300

# Parametros del transistor 100 MHz
y11 =   1.0e-3 
y12 =   0.0
y21 =  30.0e-3  
y22 =   0.1e-3 
# Cifra de ruido del transistor
NF  = 2 # dB 
F   = 10**(NF/10) # Factor de ruido del transistor 


Q1  = 40                      # Q cargado del sintonizado
Qo  = 10000                   # Q libre de perdidas

rg   = 1/y11.real             # Resistencias del generador
r11  = 1/y11.real             # resistencia de entrada 
gm   = abs(y21)               # gm del transistor
r22  = 1/y22.real             # resistencia de salida
rl   = 1/y22.real             # Resistencia de carga

rext = rg/2                   # resistencia externa para el calculo del inductor
L1   = (1/Q1 - 1/Qo)*rext/wo  # Calculo del inductor
C1   = 1/(wo**2 * L1)         # Capacitor de sintonia
rp1  = Qo * wo * L1           # Resistencia de perdida

rti  = Q1*wo*L1               # resistencia total en el nodo de entrada (incluye sintonizado)
rto  = r22*rl/(r22 + rl)      # resistencia total en el nodo de salida (no tiene sintonizado)  
Av   = gm * rto               # Ganancia de tension
G    = Av**2 * r11/rl         # Ganancia de potencia
print('Ganancia de potenica: {:1.2e} V'.format(G) ) 


Beq1 = (np.pi/2)*(fo/Q1)      # Ancho de banda equivalente del sintonizado
print('Ancho de banda equivalente: {:1.2e} Hz '.format(Beq1) ) 

# Calculo de la fuentes de corriente de ruido 
iib   = (4*kb*T/rti)**(1/2) ## Corriente de ruido dadas las resistencias en la entrada 
Ni   =  iib**2* Beq1 *r11      # N Potencia de ruido de entrada
print('Potencia de ruido de entrada Ni: {:1.2e} W'.format(Ni)) 

iei    = (F - 1)**(1/2)  *  iib  ## Fuente de corriente de ruido en exceso en la entrada
# Si la fuente se encuetra en el nodo de salida 
#ieo    = ((F - 1)*G*rti/rto)**(1/2)*ii   

inoise = (iib**2 + iei**2)**(1/2)   # Corriente total en el nodo de entrada 
print('Valor cuadratico medio de la corriente sobre el resistor de entrada: {:1.2e} V'.format(inoise) ) 

No    = G * (inoise**2 * Beq1 * r11)
print('Potencia de ruido sobre el resistor de salida: {:1.2e} W'.format(No)) 

F     = No/(G*Ni)
print('F : {:1.2e} '.format(F)) 
NF    = 10 * np.log10(F)
print('NF : {:1.2e} '.format(NF)) 

Ganancia de potenica: 2.25e+03 V
Ancho de banda equivalente: 3.93e+06 Hz 
Potencia de ruido de entrada Ni: 1.31e-13 W
Valor cuadratico medio de la corriente sobre el resistor de entrada: 7.26e-12 V
Potencia de ruido sobre el resistor de salida: 4.66e-10 W
F : 1.58e+00 
NF : 2.00e+00 


In [7]:
## Parametros de LTspice
# para usar en LTspice. Presionar s y pegar en el cuadro de dialogo.
# Los valores de los componentes se asignan entre llaves. 

## Para las simulaciones con LTspice
gei = iei/1e-12


print('.param L1 = {:1.3e}'.format(L1))
print('.param C1 = {:1.3e}'.format(C1))
print('.param rp1 = {:1.3e}'.format(rp1))
print('.param rg = {:1.3e}'.format(rg))
print('.param r11 = {:1.3e}'.format(r11))
print('.param gm = {:1.3e}'.format(gm))
print('.param r22 = {:1.3e}'.format(r22))
print('.param rl = {:1.3e}'.format(rl))
print('.param gei = {:1.3e}'.format(gei))


.param L1 = 1.981e-08
.param C1 = 1.278e-10
.param rp1 = 1.245e+05
.param rg = 1.000e+03
.param r11 = 1.000e+03
.param gm = 3.000e-02
.param r22 = 1.000e+04
.param rl = 1.000e+04
.param gei = 4.411e+00


### Medición con LTspice

![13-cuatrocientos](fig/unSimpleIn.png)

Valor cuadratico medio de la tensión en el nodo de entrada: $5.5858e-6 V$

Potencia de ruido sobre el resistor $r11$: 

$$Ni = \frac{vni^2}{r11} =  3.12 \times 10^{-14} W$$

![14-cuatrocientos](fig/unSimpleOut.png)

Valor cuadratico medio de la tensión en el nodo de entrada: $1.11e-3 V$

Potencia de ruido sobre el resistor $rl$: 

$$No = \frac{vno^2}{rl} =  4.66 \times 10^{-10} W$$


## Ejemplo Dos Simples  Sintonizados empleando transistor con factor de ruido $F = 2$.

Calcular el ancho de banda equivalente de un amplificador simple sintonizado que tiene dos sintonizados. Calcular para máxima transferencia de energía. 

Datos:
$f_o  = 100 MHz$

$Q_{c1}  = 40$
$Q_{c2}  = 40$


$Q_o = inf$  

Transistor: 

$y11 =   1.0\times 10^{-3}S$,

$y12 =   0.0S$, 

$y21 =  30.0\times 10^{-3}S$, 

$y22 =   0.1\times 10^{-3}S$.

In [8]:
## Ejemplo con calculo en Python
# simple sintonizado de 100MHz
# Frecuencia de operación
fo = 100e6
wo= 2*np.pi*fo

kb = constants.value('Boltzmann constant')
# Temperatura en Kelvin
T  = 300

# Parametros del transistor 100 MHz
y11 =   1.0e-3 
y12 =   0.0
y21 =  30.0e-3  
y22 =   0.1e-3 
# Cifra de ruido del transistor
NF  = 2 # dB 
F   = 10**(NF/10) # Factor de ruido del transistor 


Q1  = 40                      # Q cargado del sintonizado
Q2  = 40                      # Q cargado del sintonizado
Qo  = 10000                   # Q libre de perdidas

rg   = 1/y11.real             # Resistencias del generador
r11  = 1/y11.real             # resistencia de entrada 
gm   = abs(y21)               # gm del transistor
r22  = 1/y22.real             # resistencia de salida
rl   = 1/y22.real             # Resistencia de carga

## Entrada
rext = rg/2                   # resistencia externa para el calculo del inductor
L1   = (1/Q1 - 1/Qo)*rext/wo  # Calculo del inductor
C1   = 1/(wo**2 * L1)         # Capacitor de sintonia
rp1  = Qo * wo * L1           # Resistencia de perdida
rti  = Q1*wo*L1               # resistencia total en el nodo de entrada (incluye sintonizado)

## Salida
rext = r22/2                  # resistencia externa para el calculo del inductor
L2   = (1/Q2 - 1/Qo)*rext/wo  # Calculo del inductor
C2   = 1/(wo**2 * L2)         # Capacitor de sintonia
rp2  = Qo * wo * L2           # Resistencia de perdida
rto  = Q2*wo*L2               # resistencia total en el nodo de salida (no tiene sintonizado)  


Av   = gm * rto               # Ganancia de tension
G    = Av**2 * r11/rl         # Ganancia de potencia
print('Ganancia de potenica: {:1.2e} V'.format(G) ) 


# Ancho de banda equivalente para las fuentes de ruido en la entrada del sintonizado
Beq1  = (np.pi/2)*(fo/Q1)      
print('Ancho de banda equivalente de la entrada: {:1.2e} Hz '.format(Beq1) )
# Ancho de banda equivalente para las fuentes de ruido en la salida del sintonizado
Beq2  = (np.pi/2)*(fo/Q2)      
print('Ancho de banda equivalente de la salida: {:1.2e} Hz '.format(Beq2) ) 

# Ancho de banda equivalente para las fuentes de ruido de la entrada del sintonizado
Beq1o = (np.pi/4)*(fo/Q1) 
print('Ancho de banda equivalente: {:1.2e} Hz '.format(Beq1o) ) 


# Calculo de la fuentes de corriente de ruido 
iib   = (4*kb*T/rti)**(1/2) ## Corriente de ruido dadas las resistencias en la entrada 
Ni    =  iib**2* Beq1o *r11      # N Potencia de ruido de entrada
print('Potencia de ruido de entrada Ni: {:1.2e} W'.format(Ni)) 

iei    = (F - 1)**(1/2)  *  iib  ## Fuente de corriente de ruido en exceso en la entrada
# Si la fuente se encuetra en el nodo de salida 
ieo    = ((F - 1)*G*rti/rto)**(1/2)*iib   

inoise = (iib**2 + iei**2)**(1/2)   # Corriente total en el nodo de entrada 
print('Valor cuadratico medio de la corriente sobre el resistor de entrada: {:1.2e} A'.format(inoise) ) 

No    = G * (inoise**2 * Beq1o * r11)
print('Potencia de ruido sobre el resistor de salida: {:1.2e} W'.format(No)) 

F     = No/(G*Ni)
print('F : {:1.2e} '.format(F)) 
NF    = 10 * np.log10(F)
print('NF : {:1.2e} '.format(NF)) 

Ganancia de potenica: 2.23e+03 V
Ancho de banda equivalente de la entrada: 3.93e+06 Hz 
Ancho de banda equivalente de la salida: 3.93e+06 Hz 
Ancho de banda equivalente: 1.96e+06 Hz 
Potencia de ruido de entrada Ni: 6.53e-14 W
Valor cuadratico medio de la corriente sobre el resistor de entrada: 7.26e-12 A
Potencia de ruido sobre el resistor de salida: 2.31e-10 W
F : 1.58e+00 
NF : 2.00e+00 


In [9]:
## Parametros de LTspice
# para usar en LTspice. Presionar s y pegar en el cuadro de dialogo.
# Los valores de los componentes se asignan entre llaves. 

## Para las simulaciones con LTspice
gei = iei/1e-12
geo = ieo/1e-12

print('.param rg = {:1.3e}'.format(rg))
print('.param L1 = {:1.3e}'.format(L1))
print('.param C1 = {:1.3e}'.format(C1))
print('.param rp1 = {:1.3e}'.format(rp1))
print('.param gei = {:1.3e}'.format(gei))
print('.param geo = {:1.3e}'.format(geo))
print('.param r11 = {:1.3e}'.format(r11))
print('.param gm = {:1.3e}'.format(gm))
print('.param r22 = {:1.3e}'.format(r22))
print('.param L2 = {:1.3e}'.format(L2))
print('.param C2 = {:1.3e}'.format(C2))
print('.param rp2 = {:1.3e}'.format(rp2))
print('.param rl = {:1.3e}'.format(rl))


.param rg = 1.000e+03
.param L1 = 1.981e-08
.param C1 = 1.278e-10
.param rp1 = 1.245e+05
.param gei = 4.411e+00
.param geo = 6.590e+01
.param r11 = 1.000e+03
.param gm = 3.000e-02
.param r22 = 1.000e+04
.param L2 = 1.981e-07
.param C2 = 1.278e-11
.param rp2 = 1.245e+06
.param rl = 1.000e+04


## Medición con LTspice

![15-cuatrocientos](fig/dosSimpleIn.png)

Valor cuadratico medio de la tensión en el nodo de entrada: $5.5858e-6 V$

Potencia de ruido sobre el resistor $r11$: 

$$Ni = \frac{vni^2}{r11} =  3.12 \times 10^{-14} W$$

![16-cuatrocientos](fig/dosSimpleOut.png)

Valor cuadratico medio de la tensión en el nodo de entrada: $757e-6 V$

Potencia de ruido sobre el resistor $rl$: 

$$No = \frac{vno^2}{rl} =  5.73 \times 10^{-11} W$$


![17-cuatrocientos](fig/dosSimpleOut2.png)

Valor cuadratico medio de la tensión en el nodo de entrada: $885e-6 V$

Potencia de ruido sobre el resistor $rl$: 

$$No = \frac{vno^2}{rl} =  7.85 \times 10^{-11} W$$


#### Nota

La diferencia entre los dos metodos radica en que la fuente de corriente de ruido en la entrada se filtra por dos sintonizados, lo que da un ancho de banda equivalente de ruido de $\frac{\pi}{4}$, en cambio, al estar en la salida, el ancho de banda equivalente es de $\frac{\pi}{2}$.

## Ancho de banda de ruido equivalente

Se define como ancho de banda equivalente de ruido, al ancho de banda que debería tener un dispositivo ideal para producir en la salida la misma potencia de ruido.

La potencia de ruido a la salida del dispositivo puede calcularse integrando la densidad espectral de potencia de ruido a la salida $N_{noise}*|H(f)|^2$ para todas las frecuencias, donde $|H(f)|^2$ es la ganancia de potencia del dispisitivo en función de la frecuencia (puede ser un amplificador, un filtro o un mezclador). Esta potencia tiene que coincidir con la potencia a la salida de un filtro ideal de ancho de banda $B_{eq}$ y ganancia igual a la ganancia en frecuencia central $f_{c}$. 

Esto demuestra que el ancho de banda se calcula de la siguiente forma:

$$B_{eq}=\frac{\int_{0}^{\infty} |H(f)|^2 df}{|H(f_{c})|^2}=\int_{0}^{\infty} \frac{|H(f)|^2}{|H(f_{c})|^2} df =\int_{0}^{\infty} \bar{|H(f)|^2} df$$

## Ruido en un receptor heterodino

La figura muestra un receptor heteronido de doble conversión. 

![18-cuatrocientos](fig/receptor.png)

El factor de ruido del receptor se puede calcular como:


$$F_{tot}=F_{1}+\frac{F_{2}- 1}{G_{1}G_{2}}+...+\frac{F_{N} -1}{\prod_{i=1}^{N-1}G_{i}}$$

Podemos aplicar la formula de Frizz, si los anchos de banda de ruido se reducen en las sucesivas etapas. 

Dado que la incidencia del factor de ruido es inversamente proporcial a las ganancias anteriores, para la mayoria de los casos prácticos, las primeras etapas son las que definen el ruido del sistema. 

Con respecto al ancho de banda de ruido equivalente, podemos analizar los anchos de banda de ruido de cada etapa para ver como inciden en el ancho de banda total del sistema. 

En este caso, por ejemplo, si la etapa de RF esta sintoniza en $100 MHz$, su ancho de banda probablemente sea de algunos $MHz$, la etapa de primera de frecuencia intermedia tiene una frecuencia de $10.7 MHz$ con un ancho de banda de banda de cientos de $KHz$ y por último, la etapa de segunda frecuencia intermedia esta sintonizada a $455 KHz$ con un ancho de banda de decenas de $KHz$. En este sistema, la cifra de ruido se define por las primeras etapas (RF y 1FI) y el ancho de banda equivalente de ruido esta definido principalmente por la etapa de segunda FI.  


## Tres simples sintonizados sincrónicos:

$$\bar{|H(f)|^2}=\frac{1}{(1+\chi^2)^3}$$

Donde: $\chi^2=\frac{2 Q}{f_{o}} \cdot (f-f_{o})$, entonces $df= \frac{f_{o}}{2Q}d\chi$. 

El ancho de banda equivalente se calcula de esta forma

$$B_{eq}=\int_{0}^{\infty} \frac{1}{(1+\chi^2)^3} \frac{f_{o}}{2Q} d\chi$$

El ancho de banda equivalente es el ancho de banda de 3dB por $\pi/2$


$$B_{eq}=\frac{3 \cdot \pi}{16} \cdot \frac{f_{o}}{2Q}$$



## Butterwoth de tercer orden

La transferencia normalizada de un Butterwoth de orden $n$ puede expresarse como: 

$$\bar{|H(f)|}=\frac{1}{\sqrt{1+(\frac{\chi(f)}{\chi(f_c)})^{2\cdot n}}}$$ 

donde:

$$\chi(f)   = 2 \cdot Q \cdot \frac{(f-f_o)}{f_o}$$

$$\chi(f_c) = 2 \cdot Q \cdot \frac{(f_c-f_o)}{f_o}$$

Esta transferencia corresponde a un filtro pasabajo, por lo tanto para el cálculo del ancho de banda equivalante podemos usar esta ecuación teniendo en cuenta que corresponde a la mitad del ancho de banda. 

Entonces:

$$B_{eq}=2 \cdot \int_{0}^{\infty} \frac{1}{1+(\frac{\chi(f)}{\chi(f_c)})^{6}} \frac{fo}{2Q}d\chi = \frac{fo}{Q} \cdot \chi(f_c) \cdot\frac{ \pi}{3} = 2 \cdot \frac{fo}{2Q} \cdot 2\cdot Q \frac{(f_c-f_o)}{f_o} \cdot \frac{ \pi}{3}$$

$$B_{eq}= 2 \cdot (f_c-f_o) \cdot \frac{ \pi}{3}$$

